x-pack/test/functional/page_objects/space_selector_page.ts

```diff
@@ -205,6 +205,7 @@ export class SpaceSelectorPageObject extends FtrService {
       this.log.info(`SpaceSelectorPage:goToSpecificSpace(${spaceId})`);
       await this.testSubjects.click(`${spaceId}-selectableSpaceItem`);
       await this.common.sleep(1000);
```

Comment:
Do we still need this artificial timeout now that we're checking for the menu to be closed?

Reply:
I was inclined to leave it in because there seems to sometimes be a delay between the click and the closing of the menu/redirect. Without it, it may be possible for another race condition resulting in the space item not even being there on the next retry.
If this doesn't solve the problem we were seeing, I think I will refactor the test to be as deterministic as possible, in which case we can take out the delay.